homeassistant/components/osc/const.py

```diff
@@ -0,0 +1,4 @@
+"""Constants for the OSC (Open Sound Control) integration."""
+
+DOMAIN = "osc"
+DEFAULT_LISTEN_PORT = 9000
```

Comment:
where is this being used? You can set it as the default for the port in the schema in `config_flow.py`